### PR TITLE
Integrate with Redfish schema

### PR DESCRIPTION
**Pull request by jzding**
This update works with the Redfish schema updates in sdk-go repo.
A successful build is depending on sdk-go repo being updated first.

Signed-off-by: Jack Ding <jacding@redhat.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,9 +5,9 @@ go 1.15
 require (
 	github.com/cloudevents/sdk-go/v2 v2.4.1
 	github.com/prometheus/client_golang v1.11.0
-	github.com/redhat-cne/rest-api v0.0.0-20210628151454-c8f37f0a9c3e
-	github.com/redhat-cne/sdk-go v0.0.0-20210630032545-4072da5111ba
+	github.com/redhat-cne/rest-api v0.1.0
```
Comment (aneeshkp) [inline go.mod:8]: you might have to use main branch commit.

File: vendor/modules.txt [hunk 2]
```diff
@@ -7,7 +7,7 @@ github.com/BurntSushi/toml
 github.com/beorn7/perks/quantile
 # github.com/cespare/xxhash/v2 v2.1.1
 github.com/cespare/xxhash/v2
-# github.com/cloudevents/sdk-go/protocol/amqp/v2 v2.3.1-0.20210302080936-5c462007a5d5
+# github.com/cloudevents/sdk-go/protocol/amqp/v2 v2.4.1
```
Comment (aneeshkp) [inline vendor/modules.txt:10]: what was introduced or fixed in v2.4.1 ?

File: vendor/modules.txt [hunk 3]
```diff
@@ -69,7 +69,7 @@ github.com/prometheus/procfs/internal/util
 ## explicit
 github.com/redhat-cne/rest-api
 github.com/redhat-cne/rest-api/pkg/localmetrics
-# github.com/redhat-cne/sdk-go v0.0.0-20210630032545-4072da5111ba
+# github.com/redhat-cne/sdk-go v0.0.0-local => ../sdk-go
```
Comment (aneeshkp) [inline vendor/modules.txt:72]: need updated

Comment (jzding) [reply vendor/modules.txt:72]: right, will update mod file once the sdk-go redfish-schema change is delivered.

File: go.mod (hunk 1 above)
Comment (jzding) [reply go.mod:8]: will update mod file once the sdk-go redfish-schema change is delivered.

File: vendor/modules.txt (hunk 2 above)
Comment (jzding) [reply vendor/modules.txt:10]: not sure. It is generated by `go mod tidy`.

File: go.mod (hunk 1 above)
Comment (aneeshkp) [reply go.mod:8]: this needs to be fixed

**Review by aneeshkp — APPROVED** [review]

**Pull request merged**